mix/mix_protocol.nim

```diff
@@ -12,13 +12,18 @@ import
 
 const MixProtocolID* = "/mix/1.0.0"
 
-type MixProtocol* = ref object of LPProtocol
+# Possibly important for reply block
+type MixProtocol* = ref object
+  lpProto: LPProtocol
```

Comment:
Composition over inheritence is almost always preferable in my personal opinion. Inheritence structures can very quickly get out of hand.